Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Invoice.hs

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS_GHC -Wwarn=unused-imports #-}
+
+module Domain.Action.UI.Invoice (getInvoice) where
+
+import qualified API.Types.UI.Invoice
+import Control.Monad (msum)
+import Data.OpenApi (ToSchema)
+import qualified Domain.Types.Merchant
+import qualified Domain.Types.MerchantOperatingCity
+import qualified Domain.Types.Person
+import qualified Environment
+import EulerHS.Prelude hiding (id)
+import qualified Kernel.Prelude
+import Kernel.Types.Distance (Meters (..), metersToHighPrecMeters)
+import Kernel.Types.Error (PersonError (PersonNotFound))
+import qualified Kernel.Types.Id
+import Kernel.Utils.Common (fromMaybeM)
+import Servant
+import qualified Storage.Clickhouse.Ride as CHR
+import qualified Storage.Clickhouse.RideDetails as CHRD
+import qualified Storage.Queries.Person as QP
+import Tools.Auth
+
+getInvoice ::
+  ( Kernel.Prelude.Maybe (Kernel.Types.Id.Id Domain.Types.Person.Person),
+    Kernel.Types.Id.Id Domain.Types.Merchant.Merchant,
+    Kernel.Types.Id.Id Domain.Types.MerchantOperatingCity.MerchantOperatingCity
+  ) ->
+  Kernel.Prelude.Maybe (Kernel.Prelude.UTCTime) ->
```

Comment:
`, run-generator  -- --apply-hint --path=Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Invoice.hs`

this will remove redundant brackets